tools/gulptasks/dist-productsjs.js

```diff
@@ -42,22 +43,33 @@ function distProductsJS() {
             ''
         );
 
-        const nr = (
-            buildProperties.version ||
-            packageJson.version ||
-            ''
-        );
+        function getNr(name) {
+            return (
+                (name && buildProperties.products[name].version) ||
+                buildProperties.version ||
+                packageJson.version ||
```

Comment:
This will not work as Highcharts is not our package. Use `argv.release` for dashboards.